en/docs/guides/workflows/add-a-new-workflow-definition.md

```diff
@@ -1,78 +0,0 @@
-# Add a New Workflow Definition
```

Comment:
instead of deleting the `workflow` related files, can we hide the access to those docs?